doc/plugins/prometheus.md

```diff
@@ -122,12 +122,56 @@ Or you can goto [Grafana official](https://grafana.com/grafana/dashboards/11719)
 
 ### Available metrics
 
-* `Status codes`: HTTP status codes returned by upstream services. These are available per service and across all services.
+* `Status`: HTTP status codes returned by upstream services. These are available per service and across all services.
+
+    Attributes：
+
+    | Name         | Description         |
+    | -------------| --------------------|
+    | code         | The HTTP status code returned by the upstream service. |
+    | route        | The route_id of the request to match the route. If it does not match, the default is `""`. |
+    | matched_uri  | Request to match the uri of the route. If it does not match, the default is `""`. |
+    | matched_host | Request to match the hsot of the route. If it does not match, the default is `""`. |
+    | service      | The request matches the service_id of the route. If it does not match, the default is host. |
```

Comment:
"host" should be `$host`? And we use the default value not because "it does not match", but because there is no service id of the matched route.

Reply:
Yes, the description here is inappropriate.

Reply:
updated.